pkgs/by-name/db/dbdev/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  SystemConfiguration,
+  fetchFromGitHub,
+  gitUpdater,
+  openssl,
+  pkg-config,
+  rustPlatform,
+  stdenv,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "dbdev";
+  version = "0.1.6";
+  sourceRoot = "${src.name}/cli";
+
+  src = fetchFromGitHub {
+    owner = "supabase";
+    repo = "dbdev";
+    rev = "v${version}";
+    hash = "sha256-G9U+dhSJI03n0lbBEVOrzslj8TlmiRwrg8DQ7hGsqIo=";
+  };
+
+  cargoHash = "sha256-v7QHUCKJjfTiG8ZylBmJ0rWRaeU9imGJBQhcESzZyUM=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  # Needed to get openssl-sys to use pkg-config.
+  OPENSSL_NO_VENDOR = 1;
```

```suggestion
  env.OPENSSL_NO_VENDOR = 1;
```

We are slowly moving to set environment variables with `env.VAR`, so it would be good to do it with here in a new package also.